dev_guide/managing_images.adoc

```diff
@@ -0,0 +1,690 @@
+= Managing Images
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+An
+link:../architecture/core_concepts/builds_and_image_streams.html#image-streams[image
+stream] comprises any number of
+link:../architecture/core_concepts/containers_and_images.html#docker-images[Docker
+images] identified by tags. It presents a single virtual view of related images,
+similar to a Docker image repository.
+
+By watching an image stream, builds and deployments can receive notifications
+when new images are added and react by performing a build or deployment,
```

Comment:
I'd say added or modified, although each push is a new image (in certain sense) for a user create/update is more natural, imho.